roles/include_components/README.md

```diff
@@ -1,15 +1,46 @@
-# include_components role
+# redhat.ocp.include_components role - create and attach DCI components to the DCI jobs
 
-The `include_components` role adds components in the DCI jobs from software resources available in the ansible controller node, for example: rpm packages, and git repositories used during the job execution.
+- [Synopsis](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#synopsis)
+- [Parameters](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#parameters)
+- [Examples](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/include_components/README.md#examples)
```

Comment:
I wonder if you could use relative paths or the anchors to make it work in both places? :thinking:

Reply:
I tried that when I was writing documentation for the [deprecated_api](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/deprecated_api/) role. Nothing worked, and in the documentation of others, I only found the full links.